package.json

```diff
@@ -68,6 +68,7 @@
     ]
   },
   "devDependencies": {
+    "@babel/preset-typescript": "^7.24.1",
```

Comment:
Is this necessary? What are we using Babel for?

Reply:
In the AppBar.test.tsx file I am importing the Jest testing library so I can mock the minidenticons function. In the Jest documentation about using Jest with Typescript the docs state Jest supports Typescript via Babel. I followed the instructions in the official Jest docs to set this up. https://jestjs.io/docs/getting-started#using-typescript